recipes/keras-resnet/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "keras-resnet" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "8ce27ba782d1b45b127af51208aefdceb2de8d2c54646bac5fc786506ce558c0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
```

There is no need to use variables for single use values:
```suggestion
  sha256: 8ce27ba782d1b45b127af51208aefdceb2de8d2c54646bac5fc786506ce558c0
```